main.rb

```diff
@@ -0,0 +1,179 @@
+module Enumerable
+  def my_each
+    i = 0
+    length.times do
+      yield (self[i])
+      i += 1
+    end
+  end
```

Comment:
should have working `my_each` **(covered by automated tests)** identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
* my_each returns an Enumerator if no block is given

              [1,2,3].each
              #=> #<Enumerator: [1, 2, 3]:each>

             [1,2,3].my_each
             #=> [1, 2, 3]